guidelines/index.html

```diff
@@ -78,1254 +106,1196 @@ <h3>Section status levels</h3>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Obscuring primary content</h5>
+						<p class="provision-text">Advertising and other third-party content that obscures the primary content can be moved or removed without interacting with the advertising or third-party content.</p>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>
+				</section>
 
-				<p>Procedures used in assertions may be implemented at the organization level, during design and development, or during testing.</p>
+				<section class="guideline">
+					<h4>Security and Privacy</h4>
+					<p class="guideline-text">Users’ safety, security or privacy are not decreased by accessibility measures. </p>
+					<section class="provision" data-status="exploratory">
+						<h5>Clear agreement</h5>
+						<p class="provision-text">The interface indicates when a user is entering an agreement or submitting data.</p>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Disability information privacy</h5>
+						<p class="provision-text">Disability information is not disclosed to or used by third parties and algorithms (including AI).</p>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Sensitive information</h5>
+						<p class="provision-text">Prompts to hide and remove sensitive information from observers are available.</p>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>	
+					<section class="provision" data-status="exploratory">
+						<h5>Risk statements</h5>
+						<p class="provision-text">Clear explanations of the risks and consequences of choices, including use, are stated.</p>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>
+				</section>
 
-				<p>Examples of procedures that may be used during implementation might include:</p>
-					<ul><li>Training,</li>
-					<li>FTE (Full Time Equivalent) assignments,</li>
-					<li>Skills testing,</li>
-					<li>Coordination and documentation of accessibility processes, or</li>
-					<li>Setting the priority for remediation.</li></ul> 
+				<section class="guideline">
+					<h4>Algorithms</h4>
+					<p class="guideline-text">Users are not disadvantaged by algorithms.</p>
+					<section class="provision" data-status="exploratory">
+						<h5>Algorithm bias</h5>
+						<p class="provision-text">Algorithms (including AI) used are not biased against people with disabilities.</p>
```

Comment:
I feel it's too unclear what we mean by “AI” here, it seems to refer to LLMs but they are not really algorithms?

Reply:
I think it is being as general as possible. I agree it needs refinement, but a sub-group will be formed to work on this, we're just looking for any blockers for this publication.